Update webpack.config.js

xRahul opened the pull request:
Added webpack optimizations

xRahul committed:
Update webpack.config.js

Added webpack optimizations

xRahul commented:
Related Issue- https://github.com/porthos-rpc/porthos-dashboard/issues/2

gfronza commented on client/webpack.config.js:
```diff
@@ -10,6 +10,16 @@ var config = {
+        'NODE_ENV': JSON.stringify('production')
+      }
+    })
+  ]
```
missing a `,` here.

gfronza commented on client/webpack.config.js:
```diff
@@ -10,6 +10,16 @@ var config = {
     filename: 'bundle.js'
   },
+  plugins: [
+    new webpack.optimize.UglifyJsPlugin(),
```
can you add these options to prevent tons of 3rd party libs warning?

```
{
        compress: {
            warnings: false
        }
    }
```

xRahul replied:
added that and other things to optimize the uglyjs output of this plugin

gfronza reviewed (changes requested):
Thanks for your contribution. I was thinking, for development, it would be better if the uglify is not present. We could add a npm script to build with the optimization. What do you think? Thanks

xRahul committed:
Split webpack config into production and development

Created 2 different web pack configs for building the production and
the development bundle.js
For Production it will optimise the code and reduce the file size
Added command nom start as alias to nom run watch for development

xRahul committed:
removed empty plugins from dev web pack

xRahul commented:
- Split webpack config into production and development
- Created 2 different web pack configs for building the production and
  the development bundle.js
- For Production it will optimise the code and reduce the file size
- Added command nom start as alias to nom run watch for development

gfronza commented:
@xRahul That's really great! Thanks again!
